main/python3/APKBUILD

```diff
@@ -50,7 +50,6 @@ build() {
 		--with-system-expat \
 		--with-system-ffi \
 		--with-threads \
-		--with-ensurepip=install \
 		
 
 	make EXTRA_CFLAGS="$CFLAGS" 
```

Comment:
I think the commit should be squashed with the previous (`git rebase -i` and set this to `fixup`)